app/controllers/users.js

```diff
@@ -155,20 +159,83 @@ exports.signUp = (req, res) => {
   }
 };
 
+exports.sendCredentials = (req, res) => {
+  if (req.params.token) {
+    User.findOne(
+      {
+        temporaryToken: req.params.token
+      },
```

Comment:
Hi, @ddouglasz I think we can improve on this, so much so that we won't have to save the token to the `db`. Once we receive the token from the email verification route we decode it, we find the `user` and update the `active` field of that user to `true`